### DISCUSSION TITLE
Image list in portainer

### DISCUSSION THREAD

**Discussion by HolgerM1** [vogler/free-games-claimer#79] [Q&A] [answered]
Hi,
maybe somebody can comment on this. Since running the script I recognized a growing number of large images all called <html><body>

  | ghcr.io/vogler/free-games-claimer: none 
-- | --

</body>
</html>
There are about 20 files with different ID's, each about 1 GB?!
Can I delete then?

**Comment by vogler** [comment, marked as answer]
Yes, on every update it pulls a new image. They don't really occupy 20*1GB since only changed layers (most often .js files) are stored and others deduplicated.
Here's how to remove unused dangling images:
- https://forum.openmediavault.org/index.php?thread/31199-automatically-remove-unused-images-from-portainer/
- https://github.com/portainer/portainer/issues/3490

**Reply by HolgerM1** [reply]
Thank you!
